Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -453,6 +489,18 @@ void EnhancementsWidget::AddDescriptions()
       "scaling filter selected by the game.<br><br>Any option except 'Default' will alter the look "
       "of the game's textures and might cause issues in a small number of "
       "games.<br><br><dolphin_emphasis>If unsure, select 'Default'.</dolphin_emphasis>");
+  static const char TR_OUTPUT_RESAMPLING_DESCRIPTION[] = QT_TR_NOOP(
+      "Affects how the game output image is upscaled or downscaled to the window resolution.<br>"
+      "\"Default\" will rely on the GPU internal bilinear sampler which isn't gamma corrected."
+      "<br>\"Bilinear\" (gamma corrected) is a good compromise between quality and performance."
+      "<br>\"Bicubic\" is smoother than \"Bilinear\"."
+      "<br>\"Hermite\" might offer the best quality when upscaling,"
+      " at a slightly bigger perform cost.<br>\"Catmull-Rom\" is best for downscaling."
+      "<br>\"Nearest Neighbor\" doesn't do any resampling, select if you like a pixelated look."
+      "<br>\"Sharp Bilinear\" works best with 2D games at low resolutions, use if you like a sharp"
```

Comment:
"sharp look" is a really bad way to put this. This is the option users should take if they want a pixelated look, not nearest neighbor.

And since it's all about pixel clarity, it provides basically zero benefits at higher resolutions. Saying it is a "sharp look" will make users think it's a sharpening filter! This needs to be changed.

Reply:
Please write down a full description I could use for it, I didn't investigate much the use cases for the sharp bilinear filtering.
Whatever you come up with, I will use, I don't particularly care about it.

Reply:
Um, I'll do something off the top of my head here. References are https://github.com/dolphin-emu/dolphin/pull/9860 and https://dolphin-emu.org/blog/2023/05/21/dolphin-progress-report-february-march-april-2023/#new-post-processing-shaders

> Sharp Bilinear offers upscaling with sharp rendered pixels while still maintaining consistent pixel sizes. Use with native internal resolution for a pixelated look.